pcdet/utils/common_utils.py

```diff
@@ -4,8 +4,8 @@
 import random
 import shutil
 import subprocess
-import SharedArray
 
+# import SharedArray
```

Comment:
Are we getting rid of the infamous warning? 😄

Reply:
Yes, it's already like this in the latest OSS version!